packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx

```diff
@@ -188,10 +185,17 @@ const DashboardNewsroomBase: React.FunctionComponent<DashboardNewsroomProps> = p
             Use WordPress? You can download and install the Story Boost plugin to automatically embed Story Boosts on
```

Comment:
This paragraph gives context to the instructions in the following paragraph, I'm going to uncomment it in my incoming PR #1570